src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void mainApp(String[] args) {
 
+        Machine[] machines = new Machine[]
```

Comment:
![image](https://github.com/mate-academy/jv-oop/assets/20326858/78d3de95-7eb7-4124-9ad9-49988a7f1263)

Reply:
It wos to hard to pass all cheks so i add some empty lines till it pass)))